db/queries/application/queries.py

```diff
@@ -178,6 +183,23 @@ def get_count_by_status(round_ids: Optional[list] = [], fund_ids: Optional[list]
     return results
 
 
+def create_qa_base64file(application_data: dict, with_questions_file):
```

Comment:
Can we have a docstring in this method please?

Reply:
fixed